atem_dissector.lua

```diff
@@ -2226,85 +2232,95 @@ function atem_proto.dissector(tvbuf,pktinfo,root)
+			cmd_csbd_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_direction"], tvbuf:range(pos+8, 2))
+			cmd_csbd_setmask22_tree:add(pf_fields["pf_flag_cmd_cssc_setmask22_altitude"], tvbuf:range(pos+8, 2))
+			cmd_tree:add(pf_fields["pf_field_ssrc_id"], tvbuf:range(pos+10, 1))
+			cmd_tree:add(pf_fields["pf_field_borderenabled"], tvbuf:range(pos+11, 1))
+			cmd_tree:add(pf_fields["pf_field_borderbevel"], tvbuf:range(pos+12, 1))
+			cmd_tree:add(pf_fields["pf_field_padding"], tvbuf:range(pos+13, 1))
+			cmd_tree:add(pf_fields["pf_field_borderouterwidth"], tvbuf:range(pos+14, 2))
+			cmd_tree:add(pf_fields["pf_field_borderinnerwidth"], tvbuf:range(pos+16, 2))
+			cmd_tree:add(pf_fields["pf_field_borderoutersoftness"], tvbuf:range(pos+18, 1))
+			cmd_tree:add(pf_fields["pf_field_borderinnersoftness"], tvbuf:range(pos+19, 1))
+			cmd_tree:add(pf_fields["pf_field_borderbevelsoftness"], tvbuf:range(pos+20, 1))
+			cmd_tree:add(pf_fields["pf_field_borderbevelposition"], tvbuf:range(pos+21, 1))
+			cmd_tree:add(pf_fields["pf_field_borderhue"], tvbuf:range(pos+22, 2))
+			cmd_tree:add(pf_fields["pf_field_bordersaturation"], tvbuf:range(pos+24, 2))
+			cmd_tree:add(pf_fields["pf_field_borderluma"], tvbuf:range(pos+26, 2))
+			cmd_tree:add(pf_fields["pf_field_lightsourcedirection"], tvbuf:range(pos+28, 2))
+			cmd_tree:add(pf_fields["pf_field_lightsourcealtitude"], tvbuf:range(pos+30, 1))
+			cmd_tree:add(pf_fields["pf_field_padding"], tvbuf:range(pos+31, 1))
 		elseif (cmd_name == "SSBP") then
-			cmd_tree:add(pf_fields["pf_field_box"], tvbuf:range(pos+8, 1))
-			cmd_tree:add(pf_fields["pf_field_enabled"], tvbuf:range(pos+9, 1))
-			cmd_tree:add(pf_fields["pf_field_inputsource0"], tvbuf:range(pos+10, 2))
-			cmd_tree:add(pf_fields["pf_field_positionx2"], tvbuf:range(pos+12, 2))
-			cmd_tree:add(pf_fields["pf_field_positiony2"], tvbuf:range(pos+14, 2))
-			cmd_tree:add(pf_fields["pf_field_size"], tvbuf:range(pos+16, 2))
-			cmd_tree:add(pf_fields["pf_field_cropped"], tvbuf:range(pos+18, 1))
-			cmd_tree:add(pf_fields["pf_field_unknown1"], tvbuf:range(pos+19, 1))
-			cmd_tree:add(pf_fields["pf_field_croptop"], tvbuf:range(pos+20, 2))
-			cmd_tree:add(pf_fields["pf_field_cropbottom"], tvbuf:range(pos+22, 2))
-			cmd_tree:add(pf_fields["pf_field_cropleft"], tvbuf:range(pos+24, 2))
-			cmd_tree:add(pf_fields["pf_field_cropright"], tvbuf:range(pos+26, 2))
+			cmd_tree:add(pf_fields["pf_field_ssrc_id"], tvbuf:range(pos+8, 1))
+			cmd_tree:add(pf_fields["pf_field_box"], tvbuf:range(pos+9, 1))
+			cmd_tree:add(pf_fields["pf_field_enabled"], tvbuf:range(pos+10, 1))
+			cmd_tree:add(pf_fields["pf_field_padding"], tvbuf:range(pos+11, 1))
+			cmd_tree:add(pf_fields["pf_field_inputsource0"], tvbuf:range(pos+12, 2))
+			cmd_tree:add(pf_fields["pf_field_positionx2"], tvbuf:range(pos+14, 2))
+			cmd_tree:add(pf_fields["pf_field_positiony2"], tvbuf:range(pos+16, 2))
+			cmd_tree:add(pf_fields["pf_field_size"], tvbuf:range(pos+16, 4))
```

Comment:
```suggestion
			cmd_tree:add(pf_fields["pf_field_size"], tvbuf:range(pos+18, 2))
```
Size is only `uint16` not `uint32` and should of course not overlap with `positiony`